releasenotes/notes/adjust-symbolic-pulses-amp-angle-representation-f5c40007416cf938.yaml

```diff
@@ -0,0 +1,25 @@
+---
+other:
+  - |
+    Qiskit Terra 0.23.0, began phasing out support of complex parameters
+    in the Pulse module. Mainly, all library symbolic pulses were converted
+    from complex amplitude representation to a duo of real (float) parameters
+    (amp, angle). To avoid problems, qiskit experiments adopted this convention.
+    
+    The ``FixedFrequencyTransmon`` library now includes an ``angle`` parameter
+    for every pulse, and the default values of ``amp`` are set to float instead
+    of complex.
+    
+    With the exception of ``HalfAngleCal``, all experiments should continue
+    function as they did before (even with complex ``amp``).
+    ``HalfAngleCal`` was changed from updating
+    the complex amplitude of the pulse, to updating the angle. When used with
+    the ``FixedFrequencyTransmon`` library, it will continue to work seamlessly 
+    in the new representation. However, when the experiment is used with custom
```

Comment:
The instruction for `HalfAngleCal` must be moved to standalone upgrade section. Also you need another bullet in the upgrade section to explain new library pulse convention.